cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-basic/management-credential-step/ManagementCredentials.tsx

```diff
@@ -190,11 +200,15 @@ function ManagementCredentials(props: Partial<StepProps>) {
                 />
             )}
             <CdsFormGroup layout="vertical-inline" control-width="shrink">
-                <div cds-layout="p-t:lg">
-                    <CdsButton onClick={handleConnect} disabled={connected || !awsState[STORE_SECTION_FORM].REGION}>
+                <div cds-layout="p-t:lg" className="aws-button-container">
+                    <CdsButton
+                        onClick={handleConnect}
+                        disabled={status === CONNECTION_STATUS.CONNECTED || !awsState[STORE_SECTION_FORM].REGION}
+                    >
                         <CdsIcon shape="connect" size="md"></CdsIcon>
-                        {connected ? 'CONNECTED' : 'CONNECT'}
+                        {status === CONNECTION_STATUS.CONNECTED ? 'CONNECTED' : 'CONNECT'}
```

Comment:
I thought our standard was to deactivate the button when connected (and not change the text)

Reply:
fixed.